doc/HACKING/ReleasingTor.md

```diff
@@ -142,33 +146,47 @@ new Tor release:
    master, merge it with "-s ours" to avoid a needless version bump.
 
 2. Make distcheck, put the tarball up in somewhere (how about your
-   homedir on your homedir on people.torproject.org?) , and tell `#tor`
-   about it. Wait a while to see if anybody has problems building it.
-   (Though jenkins is usually pretty good about catching these things.)
+   homedir on your homedir on people.torproject.org?) , and tell `#tor-dev`
+   about it.
+
+   If you want, wait until at least one person has built it
+   successfully.  (We used to say "wait for others to test it", but our
+   CI has successfully caught this kind of errors for the last several
+   years.)
+
+
+3. Make sure that the new version is recommened in the latest consensus.
+   (Otherwise, users will get confused when it complains to them
+   about its status.)
```

Comment:
This part is not that straight forward since few people have access to the dirauth conf git repository. And there has been an issue some days ago about how to do that. Since 90% of the time, weasel push the changes there for which he mentioned that he would like a request from someone from the network team.

Meaning, that section should include *how* to do that and for now it simply seems "Ask weasel to add the version XXX".

Reply:
d336b62fd9d811c72ec8e5929804039285616102 tries to improve this